src/Uno.UI/UI/Xaml/Documents/TextFormatting/FontDetailsCache.skia.cs

```diff
@@ -38,8 +38,8 @@ internal static class FontDetailsCache
 
 		SKTypeface GetDefaultTypeFace()
 		{
-			// if Segoe UI is not found, it will automatically return a system default
-			return SKTypeface.FromFamilyName(FeatureConfiguration.Font.DefaultTextFontFamily, skWeight, skWidth, skSlant);
+			return SKTypeface.FromFamilyName(FeatureConfiguration.Font.DefaultTextFontFamily, skWeight, skWidth, skSlant)
+				?? SKTypeface.FromFamilyName(null, skWeight, skWidth, skSlant);
```

Comment:
You can not use the `skWeight` and `skWidth` and `skSlant` to get the fallback default font typeface. Because it may return null when the argument is not the default value.

I test this code on Kylin system and get the `skTypeFace is null` output:

```csharp
using SkiaSharp;

var skTypeFace = SKTypeface.FromFamilyName(null, SKFontStyleWeight.ExtraBold, SKFontStyleWidth.Normal, SKFontStyleSlant.Italic);

if (skTypeFace is null)
{
    Console.WriteLine($"skTypeFace is null");
}
```

You can find my code in https://github.com/lindexi/lindexi_gd/tree/a98d68fc916df2bb89dd156f04321f2de58a75c6/SkiaSharp/HikucarwalllereWeajuyanowhea

Reply:
@lindexi Thanks for testing. I added an extra level of fallback to just `SKTypeface.FromFamilyName(null)`

Reply:
@Youssef1313 Thank you and I download the NugetPackages artifacts from https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=130429&view=artifacts&pathAsName=false&type=publishedArtifacts and test pass. It can work well and show the text.

Reply:
Maybe we can look into splitting Uno.UI artifacts from Uno.WinUI artifacts. That should get the size to the half roughly.